_sass/minima/_layout.scss

```diff
@@ -129,44 +152,44 @@
 }
 
 .footer-col {
-  float: left;
+  float: none;
```

Comment:
this `float: none` is being overridden media queries at `$on-medium` and beyond.
on `$on-small` this would bring the social-icons column vertically between the author and description, If that's the intention, leaving out the `float` is better than `float: none`